roles/pulp-workers/defaults/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+pulp_config_dir: "/etc/pulp"
+pulp_settings_file: "{{ pulp_config_dir }}/settings.py"
+pulp_group: pulp
```

Comment:
The role's README.md needs an update about the loose coupling. See pulp-content/README.md's "Shared variables"